ThunderCloud/CircleProgressView.swift

```diff
@@ -28,13 +29,25 @@ open class CircleProgressView: UIView {
         }
     }
     
-    /// Apply default shadow configuration
+    /// Apply shadow configuration based on `shadowComponents` property
     public var hasShadow = true {
```

Comment:
Think we can remove this and just use nullability of `ShadowComponents` ? 😃

Reply:
Ahh I see your last comment, breaking change... Hmm would just have to handle accordingly in `updateShadow` as you have.

Reply:
See my comment on your original comment, the only problem is if we do that it's technically a breaking change... and I'm not a massive fan of doing a major release for something so small after we've literally just made it to `v3.0.1` 😂

Reply:
Though as it's `CircleProgressView` wonder if anyone is using `hasShadow`...

Reply:
But yeah, more than happy to leave 😄

Reply:
Probably nobody, but I want to be strict on ourselves so if we do move to semantic versioning in the future then we're well-practiced at not causing ourselves any headaches :D